src/main/java/com/softserveinc/Task02.java

```diff
@@ -34,6 +34,12 @@
     public static void main(String[] args) {
         Scanner scanner = new Scanner(System.in);
         // TODO: Write your code here
+        System.out.println("How many cups of coffee you will need:");//I chose number 25
+        String number = scanner.nextLine();
+        System.out.println("For " + number + " you will need:");
+        System.out.println("5000 ml of water\n" +
```

Comment:
It is wrong. You should calculate how much water, milk, and coffee beans need for the preparation number of cups of  coffee